tests/filecheck/mlir-conversion/llvm_test.xdsl

```diff
@@ -1,29 +1,30 @@
+// RUN: xdsl-opt -t mlir | xdsl-opt -t mlir -f mlir | filecheck %s
 // RUN: xdsl-opt -t mlir --use-mlir-bindings %s | filecheck %s
 
 
 builtin.module() {
   // Type tests
-  func.func() ["sym_name" = "struct_to_struct", "function_type" = !fun<[!llvm.struct<"", [!i32]>], [!llvm.struct<"", [!i32]>]>, "sym_visibility" = "private"] {
-  // CHECK: func private @struct_to_struct(%arg0: !llvm.struct<(i32)>) -> !llvm.struct<(i32)> {
-    ^0(%0 : !llvm.struct<"", [!i32]>):
-      func.return(%0 : !llvm.struct<"", [!i32]>)
-  // CHECK-NEXT:   return %arg0 : !llvm.struct<(i32)>
+  func.func() ["sym_name" = "struct_to_struct", "function_type" = !fun<[!llvm.struct<(!i32)>], [!llvm.struct<(!i32)>]>, "sym_visibility" = "private"] {
+  // CHECK: func private @struct_to_struct(%{{.*}}: !llvm.struct<(i32)>) -> !llvm.struct<(i32)> {
+    ^0(%0 : !llvm.struct<(!i32)>):
+      func.return(%0 : !llvm.struct<(!i32)>)
+  // CHECK-NEXT:   return %{{.*}} : !llvm.struct<(i32)>
   }
   // CHECK-NEXT: }
 
-  func.func() ["sym_name" = "struct_to_struct2", "function_type" = !fun<[!llvm.struct<"", [!i32, !i32]>], [!llvm.struct<"", [!i32, !i32]>]>, "sym_visibility" = "private"] {
-    // CHECK: func private @struct_to_struct2(%arg0: !llvm.struct<(i32, i32)>) -> !llvm.struct<(i32, i32)> {
-    ^1(%1 : !llvm.struct<"", [!i32, !i32]>):
-      func.return(%1 : !llvm.struct<"", [!i32, !i32]>)
-    // CHECK-NEXT: return %arg0 : !llvm.struct<(i32, i32)>
+  func.func() ["sym_name" = "struct_to_struct2", "function_type" = !fun<[!llvm.struct<(!i32, !i32)>], [!llvm.struct<(!i32, !i32)>]>, "sym_visibility" = "private"] {
+    // CHECK: func private @struct_to_struct2(%{{.*}}: !llvm.struct<(i32, i32)>) -> !llvm.struct<(i32, i32)> {
+    ^1(%1 : !llvm.struct<(!i32, !i32)>):
```

Comment:
```suggestion
    ^1(%{{.*}} : !llvm.struct<(!i32, !i32)>):
```

Dito for later ones.

Reply:
Ahah, I made this exact mistake when changing the test case, but here this SSA value is the value we are parsing, and not the value in a CHECK ;)

Reply:
Ah yes true. I guess I haven't read enough file checks with inlined checking :sweat_smile: